lib/hyrax/configuration.rb

```diff
@@ -271,6 +271,12 @@ def arkivo_api?
       @arkivo_api ||= false
     end
 
+    attr_writer :realtime_notifications
+    def realtime_notifications?
+      return @realtime_notifications unless @realtime_notifications.nil?
+      @realtime_notifications = true
```

Comment:
@mjgiarlo would the two lines above be simpler written as `@realtime_notifications ||= true`

It would also match existing code.

Reply:
Hi @cam156! Yes, that would be simpler, but if `@realtime_notifications` is `false`, look at what happens:

```
[1] pry(main)> @realtime_notifications = false
=> false
[2] pry(main)> @realtime_notifications ||= true
=> true
[3] pry(main)> @realtime_notifications
=> true
```

:crying_cat_face:

Reply:
@cam156 Mike is following the patterns in the configuration for boolean fields.

Reply:
Ruby needs a `?=` (or something; I just made that up) operator that assigns only if the left side is `nil`.